### DISCUSSION TITLE
Multiperson filter

### DISCUSSION THREAD

**Discussion by MirasMNC** [microsoft-search/pnp-modern-search#3994] [General]
Hi,

I have a list where I use the column "Person" where I have multiple choice. I would like to use this column for filtering and unfortunately it does not work. Each filter option is like text string with all option and not each filter per person as you can see on the screen. I mapped the column on RefinableString. I have tested the same for the column "Choice" with multiple option and it works well. Am I doing something wrong?

![multiperson](https://github.com/user-attachments/assets/1dfa3ecd-9359-426a-9808-664704754880)

**Comment by wobba** [comment]
This is just how SharePoint crawled those fields. A multi-person field is stored as one string, while a multi-choice one is stored as multiple values/strings.

**Reply by MirasMNC** [reply]
Hi @wobba, thank you for your response. So if I understand it right there is no way how to solve this issue? I supposed it can be the same as multiple values in choice, termstore and so on. I was not right.

**Comment by MirasMNC** [comment]
Duplicate (https://github.com/microsoft-search/pnp-modern-search/discussions/3582#discussioncomment-10727068)
